src/providers/JsonRpcProvider.ts

```diff
@@ -74,6 +75,21 @@ export class JsonRpcProvider {
     const nodeResponse = (await req()) as string; /* '0x153cfb1ad0' */
     return tinyBig(hexToDecimal(nodeResponse));
   }
+  /**
+   * Returns information about a specified transaction
+   * Includes additional information than what is included in the Transaction type
+   * Similar to `ethers.provider.getTransaction`, some information not included
```

Comment:
I think it's just a single request, but they do some processing after the transaction returns.

https://github.com/ethers-io/ethers.js/blob/ef1b28e958b50cea7ff44da43b3c5ff054e4b483/packages/providers/src.ts/formatter.ts#L318

Reply:
I looked into this a little bit more; ethers returns as `data` while essential-eth returns as `input`. Both have the same value. Should I alter so that they both call it `data`?